virtio-devices/src/balloon.rs

```diff
@@ -317,18 +368,30 @@ pub struct Balloon {
 
 impl Balloon {
     // Create a new virtio-balloon.
-    pub fn new(id: String, size: u64, seccomp_action: SeccompAction) -> io::Result<Self> {
-        let avail_features = 1u64 << VIRTIO_F_VERSION_1;
+    pub fn new(
+        id: String,
+        size: u64,
+        reporting: bool,
+        seccomp_action: SeccompAction,
+    ) -> io::Result<Self> {
+        let mut avail_features = 1u64 << VIRTIO_F_VERSION_1;
 
         let mut config = VirtioBalloonConfig::default();
         config.num_pages = (size >> PAGE_SHIFT) as u32;
 
+        let mut queue_sizes = QUEUE_SIZES.to_vec();
+
+        if reporting {
+            avail_features |= 1u64 << VIRTIO_BALLOON_F_REPORTING;
+            queue_sizes.push(32);
```

Comment:
So this is a different size queue to the others?

Reply:
I get the size from https://patchwork.kernel.org/patch/11377073/.
```
+    if (virtio_has_feature(s->host_features, VIRTIO_BALLOON_F_REPORTING)) {
+        s->rvq = virtio_add_queue(vdev, 32, virtio_balloon_handle_report);
+    }
```
I think this size is get from PAGE_REPORTING_CAPACITY in the Linux kernel.

I will add a const for it.